### PR TITLE
Milestone7

### PR DESCRIPTION
**Pull request by Dinma-Faith-zz**
This project is built with ;

- HTML, CSS and JavaScript
- I added a mobile menu to my portfolio
- And displayed it 'none' in my CSS
- And then, called it up with JS each time the hamburger is being clicked

Kindly drop your comments, I am always open to improvements.

Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,15 +21,19 @@
...
           <nav>
+            <span class="bars" id="barsBtn"
+              ><a href="#"><i class="fa-solid fa-bars"></i
+            ></a></span>
+
             <div class="menu-content">
               <ul>
                 <li><a href="#">Portfolio</a></li>
                 <li><a href="#">About</a></li>
                 <li><a href="#">Contact</a></li>
```
Comment (brytebee) [inline index.html]: - [ ] **Required:** You have done an amazing job. 👏 
However, the nav links when clicked should take you to the section on the webpage.

_**Hint**_ 
- Kindly listen for a click event on the links.
- And take the user to the appropriate section

OR

- Give the sections `id` attributes if they don't have any.
- pass the value to the links.
- [See this.](https://webplatform.github.io/docs/guides/html_links/)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, Team👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- no linter errors
- linter files accurately setup✔️
- good GitHub flow✔️
- hamburger correctly display and disappear on click✔️
- template adhered to✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -29,17 +29,17 @@
 
             <div class="menu-content">
               <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
```
Comment (Cathella) [inline index.html:34]: - [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu is meant to disappear. I request that you hide the mobile menu after the user selecting an option.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Dinma-Faith 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

lets-try-this-again-lauren-webber


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Dinma-Faith👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**